packages/xo-server-sdn-controller/src/index.js

```diff
@@ -150,7 +150,10 @@ class SDNController extends EventEmitter {
 
   async load() {
     // Expose method to create pool-wide private network
-    const createPrivateNetwork = this._createPrivateNetwork.bind(this)
+    const createPrivateNetwork = params => {
+      params.vni = ++this._prevVni
```

Comment:
Avoid modifying `params`, this object does not belong to you, use object spreading to create a new object with all its properties + `vni`.